.github/workflows/test-cf-worker-next.yml

```diff
@@ -0,0 +1,73 @@
+name: "Test edge function compatibility for Next.js on Cloudflare Workers"
+on: push
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        env:
+            CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+            CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+            APP_URL: supertokens-node-b95.pages.dev
+            TEST_DEPLOYED_VERSION: true
+        defaults:
+            run:
+                working-directory: examples/next/with-emailpassword
+        steps:
+            - uses: actions/checkout@v2
+            - run: echo $GITHUB_SHA
+            - run: npm install git+https://github.com:supertokens/supertokens-node.git#$GITHUB_SHA
+            - run: npm install
+            - run: npm install mocha@6.1.4 jsdom-global@3.0.2 puppeteer@^11.0.0 isomorphic-fetch@^3.0.0
+
+            # Step to update the runtime to edge to all files in app/api/ and remove pages path
+            # to avoid conflicts
+            - name: Add runtime export to API files
+              run: |
+                  rm -rf pages
```

Comment:
You can just modify the example app itself to use app dir only. I think you'd only use both during migration which is weird in an example.

Reply:
I did not know the context for keeping both so I left it as is (as the netlify example also uses it). I will make this change.